pkgs/development/python-modules/enaml/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, atom
+, ply
+, kiwisolver
+, qtpy
+, sip
+, cppy
+, bytecode
+}:
+
+buildPythonPackage rec {
+  pname = "enaml";
+  version = "0.11.2";
+
+  src = fetchFromGitHub {
+    owner = "nucleic";
+    repo = pname;
+    rev = version;
+    sha256 = "1in5qa5j96qs3gsv8yaxs1l6dbm69xhzvc0pbzg0dd9kpqxfdy1j";
+  };
+
+  # qt bindings cannot be found during tests
+  doCheck = false;
+
+  propagatedBuildInputs = [
+    atom
+    ply
+    kiwisolver
+    qtpy
+    sip
+    cppy
+    bytecode
+  ];
+
+  meta = with lib; {
+    homepage = https://github.com/frmdstryr/enaml;
```

Comment:
I guess this is supposed to be `https://github.com/nucleic/enaml`?

Reply:
Ah and could you quote all URLs? Unquoted URLs are deprecated and therefore discouraged: [RFC0045](https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md)